docs/markdown/40-reuse/00-create.md

```diff
@@ -0,0 +1,28 @@
+<!-- .slide: -->
+
+# Create an action
+
+You can create an action from scratch based on an idea or you can extract some steps from a workflow and turn it into an action.
+Not everything should be an action but anything could became an action.
+
+When creating an action is in order, the question you can ask you is
```

Comment:
`When creating an action is in order`not sure about this sentence

Reply:
Rephrase, tell me what you think of it.